common/io/SerialLockTester.cpp

```diff
@@ -45,29 +46,38 @@ class SerialLockTest: public CppUnit::TestFixture {
 CPPUNIT_TEST_SUITE_REGISTRATION(SerialLockTest);
 
 void SerialLockTest::testLock() {
-  bool r1, r2;
-  int fd1, fd2, fd3;
-  const std::string path = "serialLockTestFile";
+  int fd;
+  pid_t our_pid = getpid();
 
-  OLA_ASSERT_FALSE(ola::io::FileExists(path));
+  std::stringstream str;
+  str << "serialLockTestFile." << our_pid;
```

Comment:
Probably pedantry, but it would be nice if the file doesn't look like it's some sort of special file by ending in our PID...
```suggestion
  str << "serialLockTestFile." << our_pid << ".pid";
```